projects/conda-forge.md

```diff
@@ -33,7 +33,7 @@ Christopher J. Wright, @CJ-Wright
 
 
 ### Code of Conduct
-?
+https://numfocus.org/code-of-conduct
```

Comment:
Should be https://conda-forge.org/docs/governance.html?highlight=code%20conduct#code-of-conduct

Also note that the NumFOCUS CoC is about to change (much for the better).

Reply:
Fair point :wink: